src/main/kotlin/fi/metatavu/timebank/api/impl/PersonsApi.kt

```diff
@@ -21,6 +22,16 @@ class PersonsApi: PersonsApi, AbstractApi() {
 
     @Inject
     lateinit var personsTranslator: PersonsTranslator
+    override suspend fun findPerson(personId: Int): Response {
+        loggedUserId ?: return createUnauthorized("Invalid token!")
+        val person = personsController.findPerson(personId)
+
+        return if (person != null) {
+            createOk(entity = person)
+        } else {
+            createNotFound("Person with id $personId not found!")
+        }
+    }
```

Comment:
Given that we've deprecated the `Persons` resource in this PR, I'd say don't implement this new method. If it is included in the spec I'd love to see PR for the specs where this endpoint is removed.